README.md

```diff
@@ -106,6 +106,7 @@ client.category_and_brand #=> CategoryAndBrandResource
 
 * `client.category_and_brand.category(id: 'id')`
 * `client.category_and_brand.category_tree(levels: 'number of levels')`
+* `client.category_and_brand.brand(brand: 'name of brand')`
```

Comment:
This documentation is wrong.

The method doesn't receive any parameter as it is intended to retrieve the brand list